cmake.deps/CMakeLists.txt

```
@@ -12,13 +12,13 @@ list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake" "${PROJECT_SOURCE_DI
 include(CheckCCompilerFlag)
 include(Util)
 
+set_default_buildtype()
+
 get_property(isMultiConfig GLOBAL PROPERTY GENERATOR_IS_MULTI_CONFIG)
 if(NOT isMultiConfig)
   set(BUILD_TYPE_STRING -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE})
```

this should be useful

```suggestion
  set(BUILD_TYPE_STRING -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE})
  message(STATUS "CMAKE_CONFIGURATION_TYPES=${CMAKE_CONFIGURATION_TYPES}")
```

---

Wait, why?

---

yeah, that's why I suggested to print it inside the condition when it's multi-config

---

Sorry, I don't get it. The condition is `if(NOT isMultiConfig)`, so it won't be printed when it's multi-config?